content/en/contribute/code/workflow.md

```diff
@@ -20,7 +20,7 @@ For a more efficient feedback loop, developers should run linting scripts locall
 
 ## Issues
 
-Issues are managed in GitHub. Issues should be created in the repository where the changes need to be made. If it is not clear in which repo to open an issue the default should be the `cht-core` repository. If it is a security or sensitive issue it should be opened in the private `medic-projects` repository.
+Issues are managed in GitHub. Issues should be created in the repository where the changes need to be made. If it is not clear in which repo to open an issue the default should be the `cht-core` repository. If it is a security or sensitive issue, reach out privately to the CHT maintainers so they can open an issue in the private `medic-projects` repository.
```

Comment:
is there a list of known maintainers?

Reply:
Excellent point! I linked the [Development Team](https://github.com/orgs/medic/teams/development-team) as the maintainers of the codebases.